component/loki/source/gcplog/gcplog.go

```diff
@@ -31,13 +32,10 @@ func init() {
 // Arguments holds values which are used to configure the loki.source.gcplog
 // component.
 type Arguments struct {
-	// TODO(@tpaschalis) Having these types defined in an internal package
-	// means that an external caller cannot build this component's Arguments
-	// by hand for now.
```

Comment:
I had to use them, so made them exported. Also to avoid import cycle, I had to put them in a different package.

Reply:
Change makes sense and LGTM but will give @tpaschalis a chance to get eyes if he has time before merge